### ISSUE TITLE
Kill lodash

### ISSUE THREAD

**Issue by dxinteractive** [bigdatr/bd-stampy#46]
Remove all lodash, because lodash isn't really necessary (immutable.js partially covers it), and its usage in bd-stampy means that some projects can't use the latest bd-stampy without a huge lodash-upgrade.

Ticked when code complete and pushed to `feature/killing-lodash` branch.
- [x] Button
- [x] Code
- [x] Config
- [x] DataTable
- [x] DatePicker
- [x] Img
- [x] SelectStandard
- [x] Table
- [x] FormMixin
- [x] BrowserHistory
- [x] CSS
- [x] History
- [x] Maths
- [x] Url
- [x] UrlStore

**Comment by dxinteractive** [comment]
Waiting on pull request to be approved. Did take about 6 hours all up due to testing.

**Comment by dxinteractive** [comment]
Approved, bd-stampy now at 0.5.3. Contractor uses it but now core needs to as well, need a new task to update core to use 0.5.3 as there are a couple of breaking changes.

**Comment by glennvano** [comment]
Is this done @dxinteractive  - can we close it?

**Comment by dxinteractive** [comment]
Done, can close.
